ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/create-run.tsx

```diff
@@ -99,6 +95,8 @@ const formSchema = z.object({
   jobConfigContext: z.string().optional(),
 });
 
+export type FormValues = ReturnType<typeof formSchema.parse>;
```

Comment:
As this is exported, it should have a descriptive name like `CreateRunFormValues`.